sessions/recap-project-1/quiz-app/README.md

```diff
@@ -122,6 +122,18 @@ During **"Recap Project 1"** you can essentially "skip" this step, because you'r
 | -------- | ---------------------------- |
 | Settings | ![Add](./assets/screen7.png) |
 
+> ⚠️ It may be that you find your Vercel deployment to have its "Deployment Protection" turned on by default. You can turn that off by
+>
+> - navigating to the "Settings" tab of your project on Vercel,
+> - selecting "Deployment Protection" and
+> - clicking on the "Vercel Authentication" toggle.
+>
+> This can also be accomplished after the project has already been deployed.
+>
+> | Step     | Screen                        |
+> | -------- | ----------------------------- |
+> | Settings | ![Add](./assets/screen11.png) |
+
```

Comment:
In my account, I can only adjust that setting after deployment, not before. I'm not sure if this is always the case. If so, I would suggest moving this section to the bottom.

Reply:
True!